src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
 
+        int[] totals = calculateStatistics(fromFileName);
+        int totalSupply = totals[0];
```

Comment:
create method generateResult() and replace these logic there, create String report and provide it to writeStatistics method
```
int totalSupply = totals[0];
int totalBuy = totals[1];
int result = totalSupply - totalBuy;
```